internal/policy/policy.go

```diff
@@ -84,7 +84,7 @@ type State struct {
 // LoadState returns the State of the repository's policy corresponding to the
 // rslEntryID.
 func LoadState(ctx context.Context, repo *git.Repository, rslEntryID plumbing.Hash) (*State, error) {
-	e, err := rsl.GetEntry(repo, rslEntryID)
+	e, err := rsl.GetItem(repo, rslEntryID)
```

Comment:
Are an Entry and an EntryType the same?!?  They both got replaced with "Item"...

Reply:
No, they're not, this is fixing an earlier mistake. EntryType was supposed to be the generic type, representative of both Entry and Annotation. That's now replaced by Item. GetEntry was named incorrectly, because it can return either an Entry or an Annotation depending on what's requested. So GetEntry becomes GetItem.

Reply:
To summarize, this PR:
* renames EntryType to Item to avoid collision with Entry
* fixes some APIs that used Entry in the name when in fact they return an Item, previously EntryType

Reply:
Also note that only the last commit pertains to the terminology. I built this PR off the commits from #129, I'll rebase once that one's merged but doing it this way avoids some conflicts. This is also why it's still a draft.